src/CADL.Extension/Emitter.Csharp/src/type/CollectionFormat.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+export enum CollectionFormat {
```

Comment:
We have constant.ts. Can we put it into that file?

Reply:
It is the different type of collection format, it is enum, not a constant.

Reply:
should we have a dictionary from the enum values here to the delimiter as a constant?
Currently our delimiters might be hard-coded in the code.

Reply:
updated